### PR TITLE
feat(371): Adding documentation for setting up the UI and Store (consolidating the API docs)

### PR DESCRIPTION
**Pull request by stjohnjohnson**
- Adding docs for setting up UI and Store
 - Combining API Plugins into API docs
 - Fixing headers to support two levels of links

Related: https://github.com/screwdriver-cd/screwdriver/issues/371

### CODE REVIEW

File: docs/cluster-management/configure-api.md [hunk 1]
```diff
@@ -82,16 +94,154 @@ ecosystem:
...
+Set these environment variables:
+
+| Environment name          | Required | Default Value     | Description                          |  |
+|:--------------------------|:---------|:------------------|:-------------------------------------|:-|
+| DATASTORE_PLUGIN          | Yes      |                   | Set to `dynamodb`                    |  |
+| DATASTORE_DYNAMODB_PREFIX | No       | '' (empty string) | Prefix to add before all table names |  |
+| DATASTORE_DYNAMODB_ID     | Yes      |                   | AWS Access Key Id                    |  |
+| DATASTORE_DYNAMODB_SECRET | Yes      |                   | AWS Secret Access Key                |  |
+
+Or modify your `local.yaml`:
```
Comment (tkyi) [inline docs/cluster-management/configure-api.md]: `# config/local.yaml`

File: docs/cluster-management/configure-api.md [hunk 2]
```diff
@@ -82,16 +94,154 @@ ecosystem:
...
+| DATASTORE_PLUGIN             | Yes            |               | Set to `sequelize`                               |  |
+| DATASTORE_SEQUELIZE_DIALECT  | No             | mysql         | Can be `sqlite`, `postgres`, `mysql`, or `mssql` |  |
+| DATASTORE_SEQUELIZE_DATABASE | No             | screwdriver   | Database name                                    |  |
+| DATASTORE_SEQUELIZE_USERNAME | No for sqlite  |               | Login username                                   |  |
+| DATASTORE_SEQUELIZE_PASSWORD | No for sqlite  |               | Login password                                   |  |
+| DATASTORE_SEQUELIZE_STORAGE  | Yes for sqlite |               | Storage location for sqlite                      |  |
+| DATASTORE_SEQUELIZE_HOST     | No             |               | Network host                                     |  |
+| DATASTORE_SEQUELIZE_PORT     | No             |               | Network port                                     |  |
+
+Or modify your `local.yaml`:
```
Comment (tkyi) [inline docs/cluster-management/configure-api.md]: `# config/local.yaml`

File: docs/cluster-management/configure-api.md [hunk 3]
```diff
@@ -82,16 +94,154 @@ ecosystem:
...
 
-```bash
-$ export K8S_HOST=127.0.0.1
-$ export K8S_TOKEN=this-is-a-real-token
-$ export SECRET_OAUTH_CLIENT_ID=totally-real-client-id
-$ export SECRET_OAUTH_CLIENT_SECRET=another-real-client-secret
+#### DynamoDB
+Set these environment variables:
+
+| Environment name          | Required | Default Value     | Description                          |  |
```
Comment (tkyi) [inline docs/cluster-management/configure-api.md]: Extra `|` at the end? I'd check the other tables after this one, too
I think this entire table has an extra column as well

File: docs/cluster-management/configure-api.md (hunk 2 above)
Comment (tkyi) [reply docs/cluster-management/configure-api.md]: Couple other places but too lazy to comment on each one

**Comment by stjohnjohnson** [comment]
@tkyi good catches, resolved.

**Review by tkyi — APPROVED** [review]
LGTM

**Pull request merged**